doc/releases/changelog-0.25.0.md

```diff
@@ -175,232 +195,184 @@
   [(#2835)](https://github.com/PennyLaneAI/pennylane/pull/2835)
-
-  ```pycon
-  >>> sum_op = qml.ops.Sum(qml.RX(phi=1.23, wires=0), qml.ops.Sum(qml.RZ(phi=3.14, wires=0), qml.PauliX(0)))
-  >>> sum_op.arithmetic_depth
-  2
-  >>> simplified_op = sum_op.simplify()
-  >>> simplified_op.arithmetic_depth
-  1
-  ```
-
-* `qml.simplify` can now be used instead of using each operator's simplify method.
   [(#2854)](https://github.com/PennyLaneAI/pennylane/pull/2854)
-
-  This wrapper can also be used inside circuits:
-
- ```python
-  @qml.qnode(qml.device('default.qubit', wires=1))
-  def circuit(x, simplify=True):
-      op = qml.adjoint(qml.adjoint(qml.RX(x, wires=0)))
-      if simplify:
-          qml.simplify(op)
-      return qml.expval(qml.PauliZ(0))
-
-  print(qml.draw(circuit)(1.2, simplify=False))
-  print(qml.draw(circuit)(1.2))
-  ```
-
-  ```pycon
-  0: ──RX(1.20)††─┤  <Z>
-  0: ──RX(1.20)─┤  <Z>
-  ```
   
-* A `Prod` symbolic class is added that allows users to represent the Prod of operators.
-  [(#2625)](https://github.com/PennyLaneAI/pennylane/pull/2625)
-
-  The `Prod` class provides functionality like any other PennyLane operator. We can
-  get the matrix, eigenvalues, terms, diagonalizing gates and more.
-
-  ```pycon
-  >>> prop_op = Prod(qml.PauliX(0), qml.PauliZ(0))
-  >>> prop_op
-  PauliX(wires=[0]) @ PauliZ(wires=[0])
-  >>> qml.matrix(prop_op)
-  array([[ 0,  -1],
-         [ 1,   0]])
-  >>> prop_op.terms()
-  ([1.0], [PauliX(wires=[0]) @ PauliZ(wires=[0])])
-  ```
-
-  The `prod_op` can also be used inside a `qnode` as an observable.
-  If the circuit is parameterized, then we can also differentiate through the
-  product observable.
-
-  ```python
-  prod_op = Prod(qml.PauliZ(wires=0), qml.Hadamard(wires=1))
-  dev = qml.device("default.qubit", wires=2)
-
-  @qml.qnode(dev)
-  def circuit(weights):
-      qml.RX(weights[0], wires=0)
-      return qml.expval(prod_op)
-  ```
-
-  ```pycon
-  >>> weights = qnp.array([0.1], requires_grad=True)
-  >>> qml.grad(circuit)(weights)
-  tensor([-0.07059288589999416], requires_grad=True)
+  ```pycon 
+  >>> op = qml.adjoint(qml.adjoint(qml.RX(x, wires=0))) 
+  >>> op 
+  Adjoint(Adjoint(RX))(tensor([1.04719755, 1.57079633], requires_grad=True), wires=[0]) 
+  >>> qml.simplify(op) 
+  RX(tensor([1.04719755, 1.57079633], requires_grad=True), wires=[0]) 
   ```
-  
-  The `prod_op` can also be used inside a `qnode` as an operation which,
-  if parameterized, can be differentiated.
-
-  ```python
-  dev = qml.device("default.qubit", wires=3)
 
-  @qml.qnode(dev)
-  def circuit(theta):
-      qml.prod(qml.PauliZ(0), qml.RX(theta, 1))
-      return qml.expval(qml.PauliZ(1))
-  ```
+* A new function called `qml.equal` can be used to compare the equality 
+  of parametric operators.
+  [(#2651)](https://github.com/PennyLaneAI/pennylane/pull/2651)
 
   ```pycon
-  >>> circuit(1.23)
-  tensor(0.33423773, requires_grad=True)
-  >>> qml.grad(circuit)(1.23)
-  -0.9424888019316975
```

Comment:
Note: this output is not correct. Just mentioning in case this example would not be removed by this PR.

Reply:
Same here, removed example.